Rakefile

```diff
@@ -4,3 +4,18 @@
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
+
+# Create a new DB backup, and import it to the local Postgres DB
+task :db do
+  app = ENV["HEROKU_APP"] || "fastlane-refresher"
+  db_name = ENV["DB_NAME"] || "refresher"
+
+  puts "This script is going to drop your local database #{db_name} and fetch the database from heroku #{app}. Please confirm!"
+  STDIN.gets
```

Comment:
The user still has enough time to ctlr + c since creating the db, and downloading it takes a while